egs2/librimix/enh1/local/data.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Copyright 2020  Shanghai Jiao Tong University (Authors: Chenda Li, Wangyou Zhang)
+# Apache 2.0
+set -e
+set -u
+set -o pipefail
+
+log() {
+  local fname=${BASH_SOURCE[1]##*/}
+  echo -e "$(date '+%Y-%m-%dT%H:%M:%S') (${fname}:${BASH_LINENO[0]}:${FUNCNAME[1]}) $*"
+}
+
+help_message=$(cat << EOF
+Usage: $0 [--min_or_max <min/max>] [--sample_rate <8k/16k>]
+  optional argument:
+    [--min_or_max]: min (Default), max
+    [--sample_rate]: 8k (Default), 16k
+EOF
+)
+
+. ./db.sh
+
+
+min_or_max=max
+sample_rate=16k
+
+. utils/parse_options.sh
+
+if [ $# -ne 0 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+# train_set="tr_"${min_or_max}_${sample_rate}
+# train_dev="cv_"${min_or_max}_${sample_rate}
+# recog_set="tt_"${min_or_max}_${sample_rate}
+
+cdir=$PWD
+
+
+git clone https://github.com/JorisCos/LibriMix ./data/LibriMix
+
+
+# the simulation program will write data to wham_noie, so copy it to user directory in case of permission issues
+rsync -r -P ${WHAM} ${cdir}/data/wham_noise
```

Comment:
Maybe it is better to add the check of `${WHAM}` and `${LIBRISPEECH}`?

Reply:
done